infrastructure/repository/authzed/SpiceDbAccessRepository.go

```diff
@@ -105,12 +105,12 @@ func (s *SpiceDbAccessRepository) ModifySeats(assignedSubjectIDs []domain.Subjec
 
 	// Step 4 examine any errors
 	if err != nil {
-		glog.Errorf("Failed to write modify seats :%v", err.Error())
+		glog.Errorf("Error assigning %s / unassigning %s seats on license %s for org %s with %d of %d seats currently in use.\nInternal error: %v", assignedSubjectIDs, removedSubjectIDs, license.ServiceID, license.OrgID, license.InUse, license.MaxSeats, err.Error())
 
 		return spiceDbErrorToDomainError(err)
 	}
 
-	glog.Infof("Assigned operation :%v", result)
+	glog.Infof("Successfully assigned %s / unassigned %s seats on license %s for org %s. Current seats used: %d of %d\n Internal response: %v", assignedSubjectIDs, removedSubjectIDs, license.ServiceID, license.OrgID, assignedCount, license.MaxSeats, result)
```

Comment:
jfyi: using `assignedCount` here as the transaction was written, so that's the new `license.InUse` afterwards.